MFVNeutralino/test/predict2v_newsimpl.py

```diff
@@ -0,0 +1,175 @@
+from JMTucker.Tools.ROOTTools import *
+from JMTucker.Tools.general import *
+import pandas as pd
+
+presel_path = '/uscms_data/d2/tucker/crab_dirs/PreselHistosV27m'
+#sel_path = '/uscms_data/d3/dquach/crab3dirs/HistosV27m_moresidebands'
+sel_path = '/uscms/home/pkotamni/nobackup/crabdirs/Histos2trkmergedV27m'
+data = bool_from_argv('data')
+year = '2017' if len(sys.argv) < 2 else sys.argv[1]
+varname = 'nom' if len(sys.argv) < 3 else sys.argv[2] # use the BTV variations to compute syst shifts on pred2v
+print("variation: %s" % varname)
+
+# turn this on to get the proper pred2v stat err
+# FIXME eventually just use this in place of epred
+print_pred_n2v_propagated_stat_err = True
+
+if data:
+    fn, presel_scale = 'JetHT%s.root' % year, 1.
+else:
+    fn, presel_scale = 'background_%s.root' % year, 1.
+
+def propagate_product(x, y, ex, ey):
+    p = x * y
+    e = p * ((ex / x)**2 + (ey / y)**2)**0.5
+    return e
+
+def fb(ft,efft,frt):
+    return (ft-frt)/(efft-frt)
+
+presel_f = ROOT.TFile(os.path.join(presel_path, fn))
+sel_f = ROOT.TFile(os.path.join(sel_path, fn))
+effs = pd.read_csv('MiniTree/efficiencies/all_effs.csv',index_col='variant')
+cb_vals = pd.read_csv('One2Two/cb_vals/cb_vals.csv',index_col='variant')
```

Comment:
I think this should be `One2Two/cb_vals/cb_vals_for_predict2v.csv`

But actually, aren't the cb/cbbar values not used? Maybe it would make sense to drop them in this file, so that they don't accidentally cause us any issues?

Reply:
why is it *.csv when this is just a new version of the `predict2v.py.` 

And, yes -- teh cb/cbbar values can be dropped out completely with this new-simplified predict2v

Reply:
Oh sorry I thought you meant the whole file. The `One2Two/cb_vals/cb_vals_for_predict2v.csv` is the same as `One2Two/cb_vals/cb_vals.csv` in my local repo though but yeah to be compatible with the master repo I should change that.

Reply:
Thanks for updating it to `One2Two/cb_vals/cb_vals_for_predict2v.csv`! Maybe we leave it as that for now, and worry about doing some cleanup (by removing all reference to cb/cbbar in this file) in the future. Does that sound good?

Reply:
Sure! thanks for looking at these changes closely. The plan sounds good.